docs/fedora-messaging.rst

```diff
@@ -30,17 +30,28 @@ Options
     Path to a valid configuration file to use in place of the configuration in
     ``/etc/fedora-messaging/config.toml``.
 
+
 Commands
 ========
 
-There is a single sub-command, ``consume``, described in detail in its ow
-section below.
+There are three sub-commands, ``consume``, ``publish`` and ``record``, described
+in detail in their own sections below.
 
 ``fedora-messaging consume [OPTIONS]``
 
     Starts a consumer process with a user-provided callback function to execute
     when a message arrives.
 
+``fedora-messaging publish [OPTIONS] FILE``
+
+    Loads serialized messages from a file and publishes them to the specified
```

Comment:
could we get an example of what a `serialized message` looks like?

Reply:
I added a description down in the detailed section below. Please let me know if there's anything that isn't clear in the description.